### PR TITLE
Add reactivation resource

### PR DESCRIPTION
**Pull request by 2e3s**
Related to #450

### CODE REVIEW

File: spec/definitions/Reactivation.yaml [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+    allOf:
+      - $ref: "#/definitions/ResourceId"
+  reactivatedBy:
+    description: Who did the reactivation
+    type: string
+    default: customer
+    enum:
+      - merchant
+      - customer
+  description:
```
Comment (adamaltman) [inline spec/definitions/Reactivation.yaml]: reason may be more clear.

**Comment by 2e3s** [comment]
@adamaltman no, this was my intiative to add this field. And I decided to remove it because it is probably not in demand.

File: spec/definitions/Reactivation.yaml (hunk 1 above)
Comment (2e3s) [reply spec/definitions/Reactivation.yaml]: We have description in

Comment (2e3s) [reply spec/definitions/Reactivation.yaml]: We have "description" for a free form text in the cancellation, I made it so to be consistent

**Comment by 2e3s** [comment]
About a default value.. isn't it that the less API requires the easier it is to use?

**Comment by adamaltman** [comment]
I don't think so.  Not when it has mystery defaults.

**Comment by adamaltman** [comment]
Missing _links with rel self, and get one endpoint.

**Comment by 2e3s** [comment]
Actually, this specification is already implemented.

**Pull request merged**